specification/adp/resource-manager/Microsoft.AutonomousDevelopmentPlatform/preview/2021-02-01-preview/adp.json

```diff
@@ -0,0 +1,1190 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2021-02-01-preview",
```

Comment:
While adding new version the recommendation is to copy the existing version into new directory structure for first commit and then push new changes for easy review. Please create new branch with this recommendation for faster review.